packages/magento-product/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
   "exclude": ["**/node_modules", "**/.*/"],
-  "include": ["**/*.ts", "**/*.tsx"],
+  "include": [
+    "**/*.ts",
+    "**/*.tsx",
+    "../magento-search/components/SearchForm/ProductFiltersProSearchField.tsx",
```

Comment:
It seems like we are trying to add a file from outside of this package to this file which I assume is not possible?